gdp/column/column.py

```diff
@@ -50,261 +75,394 @@ def build_column(min_trays, max_trays, xD, xB, x_input, nlp_solver, provide_init
             m.comps, m.trays,
-            doc='pure component vapor pressure of component on tray in bar',
+            doc='pure component vapor pressure of component on tray [bar]',
             domain=NonNegativeReals, bounds=(1E-3, 5), initialize=init['Pvap'])
+    
+        # Variable related to fraction of critical temperature (1 - T/Tc)
         m.Pvap_X = Var(
             m.comps, m.trays,
             doc='Related to fraction of critical temperature (1 - T/Tc)',
             bounds=(0.25, 0.5), initialize=init['Pvap_X'])
+    
+        # Liquid molar enthalpy variable [kJ/mol]
         m.H_L = Var(
             m.comps, m.trays, bounds=(0.1, 16),
-            doc='Liquid molar enthalpy of component in tray (kJ/mol)', initialize=init['H_L'])
+            doc='Liquid molar enthalpy of component in tray [kJ/mol]', initialize=init['H_L'])
+    
+        # Vapor molar enthalpy variable [kJ/mol]
         m.H_V = Var(
             m.comps, m.trays, bounds=(30, 16 + 40),
-            doc='Vapor molar enthalpy of component in tray (kJ/mol)', initialize=init['H_V'])
+            doc='Vapor molar enthalpy of component in tray [kJ/mol]', initialize=init['H_V'])
+    
+        # Component liquid molar enthalpy in feed variable [kJ/mol]
         m.H_L_spec_feed = Var(
             m.comps, doc='Component liquid molar enthalpy in feed [kJ/mol]',
             initialize=init['H_L_spec_feed'], bounds=(0.1, 16))
+    
+        # Component vapor molar enthalpy in feed variable [kJ/mol]
         m.H_V_spec_feed = Var(
             m.comps, doc='Component vapor molar enthalpy in feed [kJ/mol]',
             initialize=init['H_V_spec_feed'], bounds=(30, 16 + 40))
-        m.Qb = Var(domain=NonNegativeReals, doc='reboiler duty (MJ/s)',
+    
+        # Reboiler duty variable [MJ/s]
+        m.Qb = Var(domain=NonNegativeReals, doc='reboiler duty [MJ/s]',
                    initialize=init['Qb'], bounds=(0, 8))
-        m.Qc = Var(domain=NonNegativeReals, doc='condenser duty (MJ/s)',
+    
+        # Condenser duty variable [MJ/s]
+        m.Qc = Var(domain=NonNegativeReals, doc='condenser duty [MJ/s]',
                    initialize=init['Qc'], bounds=(0, 8))
 
     else:
+        # Feed temperature variable, [K], within the minimum and maximum temperature
+        m.T_feed = Var(doc='Feed temperature [K]', domain=NonNegativeReals,
+                       bounds=(min_T, max_T), initialize=368)
 
-        m.T_feed = Var(
-            doc='Feed temperature [K]', domain=NonNegativeReals,
-            bounds=(min_T, max_T), initialize=368)
-        m.feed_vap_frac = Var(
-            doc='Vapor fraction of feed',
-            initialize=0, bounds=(0, 1))
-        m.feed = Var(
-            m.comps, doc='Total component feed flow [mol/s]', initialize=50)
+        # Vapor fraction of the feed, value between 0 and 1
+        m.feed_vap_frac = Var(doc='Vapor fraction of feed', initialize=0, bounds=(0, 1))
+
+        # Total component feed flow [mol/s]
```

Comment:
Make sure that you have the same units as above. It said before [kmol/s]

Reply:
Changed the units from [mol/s] to [kmol/s].